source/sass/mixins/_typography.scss

```diff
@@ -124,6 +124,12 @@
   }
 }
 
+@mixin doi() {
```

Comment:
Seems redundant? Current usage is confused due to `covert-link` where it's overriden (ie `all: inherit`) then overriden again...

These properties don't look like they're shared either, the contextual data pattern should set it's text styling, and the reference set the styling of the DOI. If values are the same, it's by coincidence.

Reply:
As discussed, the mixin shares letter-spacing and font-family. It does apply the same font size and line height but potentially this could change dependent where it is in the site.